gcs/src/gcs_group.cpp

```diff
@@ -2253,6 +2252,7 @@ gcs_group_fetch_pfs_info(
         strncpy(entries[i].host_name, node.name, WSREP_HOSTNAME_LENGTH);
```

Comment:
I think this should be explicitly null-terminated like the UUID below.  I'm more worried about this strncpy since the node.name is just a char * which points to a string of undetermined length.

Reply:
Fixed.